src/Servers/Kestrel/Transport.Quic/src/Internal/QuicTrace.cs

```diff
@@ -9,21 +9,21 @@ namespace Microsoft.AspNetCore.Server.Kestrel.Transport.Experimental.Quic.Intern
     internal class QuicTrace : IQuicTrace
     {
         private static readonly Action<ILogger, string, Exception?> _acceptedConnection =
-            LoggerMessage.Define<string>(LogLevel.Debug, new EventId(4, nameof(NewConnection)), @"Connection id ""{ConnectionId}"" accepted.");
+            LoggerMessage.Define<string>(LogLevel.Debug, new EventId(1, "AcceptedConnection"), @"Connection id ""{ConnectionId}"" accepted.");
         private static readonly Action<ILogger, string, Exception?> _acceptedStream =
-            LoggerMessage.Define<string>(LogLevel.Debug, new EventId(5, nameof(NewStream)), @"Stream id ""{ConnectionId}"" accepted.");
-        private static readonly Action<ILogger, string, string, Exception?> _connectionError =
-            LoggerMessage.Define<string, string>(LogLevel.Debug, new EventId(6, nameof(ConnectionError)), @"Connection id ""{ConnectionId}"" hit an exception: ""{Reason}"".");
-        private static readonly Action<ILogger, string, string, Exception?> _streamError =
-            LoggerMessage.Define<string, string>(LogLevel.Debug, new EventId(7, nameof(StreamError)), @"Connection id ""{ConnectionId}"" hit an exception: ""{Reason}"".");
+            LoggerMessage.Define<string>(LogLevel.Debug, new EventId(2, "AcceptedStream"), @"Stream id ""{ConnectionId}"" accepted.");
+        private static readonly Action<ILogger, string, Exception?> _connectionError =
+            LoggerMessage.Define<string>(LogLevel.Debug, new EventId(3, "ConnectionError"), @"Connection id ""{ConnectionId}"" unexpected error.");
+        private static readonly Action<ILogger, string, Exception?> _streamError =
+            LoggerMessage.Define<string>(LogLevel.Debug, new EventId(4, "StreamError"), @"Stream id ""{ConnectionId}"" unexpected error.");
```

Comment:
This isn't a change created by your PR, but your accurate rewording of this got me thinking we should log these more severely. It probably should be a warning rather than a debug log.

https://github.com/dotnet/aspnetcore/issues/29322